entity/contents/filters/filter_input.h

```diff
@@ -27,36 +32,119 @@ class Entity;
 ///
 /// Filters may decide to not evaluate inputs in situations where they won't
 /// contribute to the filter's output texture.
-class FilterInput final {
+class FilterInput {
```

Comment:
Nit: The headerdoc comments above are now out of date.

Reply:
Done.